app/assets/stylesheets/stream_element.scss

```diff
@@ -96,7 +96,17 @@
       font-size: $font-size-small;
       line-height: $font-size-small;
     }
-    .post-content p:last-of-type { margin-bottom: 0; }
+
+    .post-content {
+      .markdown-content {
```

Comment:
- Nesting should be no greater than 3, but was 4
- Selector should have depth of applicability no greater than 3, but was 4